### DISCUSSION TITLE
Can anyone log in to Outline through OIDC?

### DISCUSSION THREAD

**Discussion by aliuq** [outline/outline#6142] [Self Hosting] [answered]
I've deployed a private Outline, I'm using `Github` as the OIDC provider, I have a question about permissions, will other people be able to access my Outline and log in via Github?

I only want a few people to be able to log in to my Outline.

**Comment by tommoor** [comment]
Anyone that can sign-in with the OIDC provider could sign-in to Outline yes. In this case you'll want to enable the security setting "Require invites" or add domains you control to "allowed domains".

**Reply by aliuq** [reply]
Thanks for the reply, I don't see an "Require invites" setting in the security settings.

outline version is `v0.72.2`

![image](https://github.com/outline/outline/assets/11707934/e8bc332d-7faa-4609-b093-0bdcdc447ccb)

**Reply by aliuq** [reply]
I found "Require invites" is only avaliable for `app.getoutline.*`

https://github.com/outline/outline/blob/48d688c0a5c7fd8cabf028775aec16cb2b8c59c6/app/utils/isCloudHosted.ts#L1-L12

**Reply by tommoor** [reply, marked as answer]
Ah true, allowed domains is the way to go. Or you can purchase a paid license which includes this feature.

**Reply by aliuq** [reply]
Sorry, I have another question, if "example.com" is added to "allowed domains", does it mean that all email addresses logged in through OIDC must match "example.com" in order to log in correctly?

**Reply by tommoor** [reply]
That's correct, yep

**Reply by tommoor** [reply]
What restriction?

**Reply by JadenSWang** [reply]
sorry @tommoor I deleted my comment earlier, the restriction with regards to only allowing this setting on cloudHosted but I realized it's meant to be a paid restriction
